.github/workflows/bencher.yml

```diff
@@ -31,26 +60,51 @@ jobs:
           fetch-depth: 0
       - uses: actions/download-artifact@v4
         with:
-          name: release-binary-linux
-          path: release-binary-linux
-      - name: unPackage binary
-        run: tar -xzf release-binary-linux/target.tar.gz
+          name: ${{ inputs.profile }}-binary-${{ inputs.target }}
+          path: ${{ inputs.profile }}-binary-${{ inputs.target }}
+      # Linux and macOS uploads compressed binary, need to extract first
+      - name: unPackage binary (Linux or macOS)
+        if: ${{ inputs.target == 'linux' || inputs.target == 'macos' }}
+        run: tar -xzf ${{ inputs.profile }}-binary-${{ inputs.target }}/target.tar.gz
       - name: Setup Python
         uses: ./.github/actions/setup-python
       - name: Bootstrap dependencies
+        if: ${{ inputs.speedometer == true || inputs.dromaeo == true }}
         run: |
           sudo apt update
           sudo apt install -qy --no-install-recommends mesa-vulkan-drivers
           python3 ./mach bootstrap --skip-lints
       - uses: bencherdev/bencher@main
       - name: File size
-        run: ./etc/ci/bencher.py filesize target/release/servo --bmf-output size.json
+        if: ${{ inputs.file-size == true }}
+        run: |
+            ./etc/ci/bencher.py filesize ${{ inputs.binary-path }} ${{ inputs.target }}-${{ inputs.profile }} --bmf-output size.json
+            echo "SERVO_FILE_SIZE_RESULT=size.json" >> "$GITHUB_ENV"
+      # We'll additionally strip and measure the size of the binary when using production profile
+      - name: Install LLVM
+        if: ${{ inputs.stripped-file-size == true }}
```

Comment:
Do we need this additional input option? Couldn't just check if `profile == production`?

Reply:
For ohos binaries, stripping them with llvm directly doesn’t seem to work atm.

Reply:
What kind of errors are you encountering? I've been using the standard llvm installed on my system to strip libservoshell.so without problems.

Reply:
I suppose I'm doing it wrongly then, here's the error:

https://github.com/dklassic/servo/actions/runs/12481534299/job/34834716773#step:13:21

Reply:
Ah, We shouldn't be measuring the .hap file, we should measure the `libservoshell.so` dynamic library, since that is what we develop here. The .hap is kind of like an apk file on android. For android I would recommend the same, measure `libservoshell.so`, since that is where our code is.

Reply:
Android and OpenHarmony now have their libservoshell.so file size properly tracked, meanwhile stripped file size will be automatically measured when profile is set to Production.

I think we should fix #34564 and add `android-production-bencher` to `try_parser.py` later as this PR has been dragging on a while now.